gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoints-group/api-endpoints-group.component.ts

```diff
@@ -0,0 +1,163 @@
+
+  public ngOnDestroy(): void {
+    this.unsubscribe$.next();
+    this.unsubscribe$.complete();
+  }
+
+  public onSubmit(): void {
+    this.updateApi()
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe({
+        next: (api) => {
+          this.initializeComponent(api as ApiV4);
+          this.snackBarService.success(this.SUCCESSFUL_ENDPOINT_CONFIGURATION_SAVE_MESSAGE);
+        },
+        error: (error) => {
+          this.snackBarService.error(error.message);
+        },
+      });
+  }
+
+  /**
+   * Initialize the component
+   *
+   * @param api the API object
+   * @private
+   */
+  private initializeComponent(api: ApiV4): void {
+    this.api = api;
+    this.initialApi = this.api;
+    this.endpointGroupType = this.api.endpointGroups[this.ajsStateParams.groupIndex].type;
+
+    this.isReadOnly = !this.permissionService.hasAnyMatching(['api-definition-r']) || api.definitionContext?.origin === 'KUBERNETES';
+
+    const group = { ...this.api.endpointGroups[this.ajsStateParams.groupIndex] };
+
+    this.generalForm = new FormGroup({
+      name: new FormControl(
+        {
+          value: group?.name ?? null,
+          disabled: this.isReadOnly,
+        },
+        [
+          Validators.required,
+          Validators.pattern(/^[^:]*$/),
+          isUniqueAndDoesNotMatchDefaultValue(
+            this.api.endpointGroups.reduce((acc, group) => [...acc, group.name], []),
+            group?.name,
+          ),
+        ],
+      ),
+      loadBalancerType: new FormControl({ value: group?.loadBalancer?.type ?? null, disabled: false }, [Validators.required]),
+    });
+
+    this.configurationForm = new FormGroup({
+      groupConfiguration: new FormControl({
+        value: group?.sharedConfiguration ?? {},
+        disabled: this.isReadOnly,
+      }),
+    });
+
+    this.groupForm = new FormGroup({
+      general: this.generalForm,
+      configuration: this.configurationForm,
+    });
+
+    this.initialGroupFormValue = this.groupForm.getRawValue();
+  }
+
+  /**
+   * Update the current API object with the values from the forms on all tabs
+   *
+   * @param api the API object
+   * @private
+   */
+  private updateApiObjectWithFormData(api: ApiV4): UpdateApiV4 {
+    const updatedEndpointGroups = [...api.endpointGroups];
+    updatedEndpointGroups[this.ajsStateParams.groupIndex] = {
+      ...this.api.endpointGroups[this.ajsStateParams.groupIndex],
+      name: this.generalForm.getRawValue().name,
+      loadBalancer: {
+        type: this.generalForm.getRawValue().loadBalancerType,
+      },
+      sharedConfiguration: this.configurationForm.getRawValue().groupConfiguration,
+    };
+
+    return {
+      ...api,
+      endpointGroups: updatedEndpointGroups,
+    };
+  }
+
+  private updateApi(): Observable<Api> {
+    return this.apiService.get(this.api.id).pipe(
+      map((api: ApiV4) => this.updateApiObjectWithFormData(api)),
+      switchMap((updateApi) => this.apiService.update(this.api.id, updateApi)),
```

Comment:
```suggestion
      switchMap((updateApi) => this.apiService.update(this.api.id, this.updateApiObjectWithFormData(api))),
```
And remove the map L159